packages/mesa.rb

```diff
@@ -11,7 +11,7 @@ class Mesa < Package
     @_ver = git_hashtag[0, 7]
     version "amber-#{@_ver}"
   else
-    @_ver = '22.3.0'
+    @_ver = '22.3.1'
```

Comment:
Change to `compatibility 'aarch64,armv7l,x86_64'`?

Reply:
There's a version for i686 too though...

Reply:
Sorry, left out  the [x] i686 by accident...

Reply:
I don't see the binary here.  Was it missed?

Reply:
There is no 22.3.1 binary for i686. 22.3.1 is selected when kernel is >= 5.10, which will never happen on i686.